packages/dallah-design-system/tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "incremental": true,
     "jsx": "react-jsx",
     "paths": {
-      "@dallah/utils": ["../dallah-utils/src/"]
+      "@dallah/utils": ["../dallah-utils/src/"],
+      "@/*": ["./*"]
```

Comment:
are u willing to have this cross the repos, if yes move it another package and install it where you need

Reply:
No I will only use it on this repo

Reply:
its a package you need to install it not adding a reference to it like that
it will not committed in the build setup

install it inside your package like this `"@dallah/design-system": "workspace:*",`